RELEASE_NOTES.rst

```diff
@@ -165,11 +166,14 @@ New Features
 
 Improvements
 """"""""""""
+- Configurable health check threshold for triggerer (#33089, #33084)
+- Add documentation generation for CLI commands from executors (#33081, #33143, #33081)
```

Comment:
I think this can be in misc section?

Reply:
Just checked now, and it's actually part of AIP-51 but not marked so, and I didn't also include AIP-51 in this changelog, I wonder which other one I'm missing? I will skip this and mark them all as part of AIP-51